### ISSUE TITLE
piwik.js blocked by client side rules /piwik.$domain=~piwik.org,script

### ISSUE THREAD

**Issue by drakaar** [piwik/piwik#10144]
Some client side blocking plugins block access to scripts and paths that match certain known keywords such as "piwik".

It is possible to provide an option during setup which allows the user to pick a unique path name for their piwik install and ensure that all front end JS and includes are updated appropriately to the new name?

This would help make it more difficult for trackers to define rules which block all piwik installations.

Thanks!

**Comment by tsteur** [comment]
I will close this one as a duplicate of https://github.com/piwik/piwik/issues/7364 

I recommend you have a look here http://piwik.org/faq/how-to/faq_132/ to avoid sending tracking requests to `piwik.php` and you could basically simply copy `piwik.js` to another filename and avoid therefore using the term `piwik`
